Invalid rroar version tag (EINVALIDTAGNAME)

The version tag for `roarr` seems to have a typo: (There is both a >= and ^)

`package.json`
```
"roarr": ">=^7.0.3",
```

NPM Error:

```
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name ">=^7.0.3": Tags may not have any characters that encodeURIComponent encodes.
```

Thank you for providing this super useful and awesome library! :-)

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "serialize-error": "^8.0.0"
   },
   "peerDependencies": {
-    "roarr": ">=^7.0.3",
+    "roarr": ">=7.0.3",
     "slonik": ">=24.1.0"
   },
   "description": "Slonik SQL tag for constructing dynamic queries.",

